src/Shared/FileSystem/ManagedFileSystem.cs

```diff
@@ -24,7 +24,7 @@ private static bool ShouldUseMicrosoftIO
             get
             {
 #if !MICROSOFT_BUILD_ENGINE_OM_UNITTESTS
-                return ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_0) && NativeMethodsShared.IsWindows;
```

Comment:
For this particular one IIRC there was a VSCode/O# issue around it -- #7335. @JoeRobich, it looks like O# no longer uses `MSBUILDDISABLEFEATURESFROMVERSION` to work around the `Microsoft.IO.Redist` thing, right, so we should be good to drop the condition?

Reply:
That is correct.